src/lucky/tags/base_tags.cr

```diff
@@ -119,10 +119,14 @@ module Lucky::BaseTags
         attrs : Array(Symbol) = [] of Symbol,
         **other_options
       ) : Nil
+      boolean_attrs = build_boolean_attrs(attrs)
       merged_options = merge_options(other_options, options)
-      {{method_name.id}}(attrs, merged_options) do
-        text content
-      end
+      tag_attrs = build_tag_attrs(merged_options)
+      view << "<{{tag.id}}" << tag_attrs << boolean_attrs << ">" << HTML.escape(content.to_s) << "</{{tag.id}}>"
```

Comment:
This doesn't seem like it'd be a regression, but are the tag_attrs being escaped properly? For example, an `&` in the middle of an `a href`?